style.css

```diff
@@ -0,0 +1,103 @@
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #222;
+  height: 87px;
+  padding: 0px 20px;
+}
+
+.navbar-start {
+  display: flex;
+}
+
+.navbar-end {
+  display: flex;
+}
+
+.navbar-items {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-dropdown {
+  font-family: "Spartan", sans-serif;
+  font-size: 14px;
+  color: #ffffff;
+  text-decoration: none;
+  padding-left: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.main-logo-icon {
+  width: 180px;
+  height: 40px;
+}
+
+.main-logo {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.small-icon {
+  width: 18px;
+  height: 18px;
+}
+
+.arrow-container {
+  margin-left: 4px;
+}
+
+.free-trial-button {
+  font-weight: 700;
+  font-size: 12px;
+  letter-spacing: 0.5px;
+  padding: 12px 24px;
+  border: 1px solid#fff;
+  border-radius: 2px;
+  color: #fff;
+  background-color: #222;
+  font-family: 'Spartan', sans-serif;
+  margin-left: 20px;
+}
+
+.free-trial-button:hover {
+  /*horizontal(offset x) zero vertical(offset y) zero blur radius zero and spread radius set to 1px*/
+  box-shadow: inset 0 0 0 1px #fff;
+}
+
+.search-icon-container {
+  display: flex;
+  align-items: center;
+  padding-bottom: 2px;
+}
+
+body {
+  margin: 0;
+}
+
+h1 {
```

Comment:
A good practice would be do declare all your h1, h2, h3,... to reuse text format.

Reply:
as i only used h1 until now,do i need to declare the rest of items?

Reply:
True, you can do them later.